source/retryable-writes/retryable-writes.rst

```diff
@@ -212,11 +216,12 @@ This includes an `update`_ command where any statement in the updates sequence
 specifies a ``multi`` option of ``true`` or a `delete`_ command where any
 statement in the ``deletes`` sequence specifies a ``limit`` option of ``0``. In
 the context of the `CRUD`_ specification, this includes the ``updateMany()`` and
-``deleteMany()`` methods as well as ``bulkWrite()`` where the requests parameter
-includes an ``UpdateMany`` or ``DeleteMany`` operation. Drivers MUST NOT add a
-transaction ID to any single- or multi-statement write commands that include one
-or more multi-document write operations. Drivers MUST NOT retry these commands
-if they fail to return a response.
+``deleteMany()`` methods and, in some cases, ``bulkWrite()``. Drivers MUST NOT
+add a transaction ID to any single- or multi-statement write commands that
+include one or more multi-document write operations. Drivers MUST NOT retry
+these commands if they fail to return a response. With regard to
+``bulkWrite()``, drivers MUST evaluate eligibility for each emanating write
+command (after order and batch splitting) individually.
```

Comment:
"each emanating write command" seems odd to me, how about "each write command sent as part of the bulkWrite (after order and batch splitting) individually"?

Reply:
Sure. "emanating" just showed up on my Word of the Day calendar and I tried to shoehorn it in here.